### PR TITLE
update board background & transperant loading img

### PR DESCRIPTION
**Pull request by AadvaitT**
<sub>
By opening a pull request, I certify that I hold the intellectual property of the code I am submitting,
and I am granting the initial authors of WBO a perpetual, worldwide, non-exclusive, royalty-free, and irrevocable license to this code.
</sub>

### CODE REVIEW

**Comment by mrducky4** [comment]
What if the whiteboard capture fails? Then messageForRobotTool() clears the old captured image and leaves it white, I think. Should it also put the default background image back in?

File: client-data/js/board.js [hunk 1]
```diff
@@ -310,6 +311,32 @@ function removeWhiteboardSnapshot() {
 	if (elem) Tools.drawingArea.removeChild(elem);
 }
 
+/**
+ * update background to apple whiteboard image to avoid disappontment
+ * between white prestine background and yellowish background of the whiteboard.
+ */
+function updateWhiteboardBackground() {
```
Comment (mrducky4) [inline client-data/js/board.js:318]: It's probably better to put this in robottools.js since we want to call it from other files. But ok here for now.

**Comment by mrducky4** [comment]
When capturing and loading a new whiteboard background image into the svg element, it seems like the new image goes behind the existing "whiteboard_background" image. I would have thought we have to delete the old one while adding the new one.

Can we just use the existing function `updateWhiteboardSnapshot()` for both the real snapshot, and the default image when we don't have a real snapshot? It seems just using a different URL would do what we want.

File: client-data/tools/clear/clear.js [hunk 2]
```diff
@@ -43,6 +43,7 @@
                 Tools.drawingArea.innerHTML = '';
                 break;
         }
+        updateWhiteboardBackground();
```
Comment (mrducky4) [inline client-data/tools/clear/clear.js:46]: Does this add the whiteboard background image into the projected image from the robot projector? We don't want that.

**Comment by AadvaitT** [comment]
>

**Comment by AadvaitT** [comment]
> What if the whiteboard capture fails? Then messageForRobotTool() clears the old captured image and leaves it white, I think. Should it also put the default background image back in?

As per my experiments, I dont see board is getting cleared when whiteboard capture fails, am I missing something here?

**Pull request closed without merging**